src/app/pages/checkout-payment/payment-concardis-creditcard-cvc-detail/payment-concardis-creditcard-cvc-detail.component.html

```diff
@@ -0,0 +1,16 @@
+<!-- error messages -->
+<div *ngIf="isCvcExpired()">
+  <div>
+    Please enter the CVC:
+    <input type="text" [id]="'cvcDetails_' + paymentInstrument.id" size="10" placeholder="enter cvc" required />
```

Comment:
use the ish-input component and provide all necessary input parameter, ui and behavior will correspond to our guidelines this way

Reply:
Done.